src/geowombat/core/io.py

```diff
@@ -769,16 +770,27 @@ def to_raster(
     if 'height' not in kwargs:
         kwargs['height'] = data.gw.nrows
 
-    if 'crs' not in kwargs:
-        kwargs['crs'] = data.crs
-
     if 'transform' not in kwargs:
         kwargs['transform'] = data.gw.transform
 
     if 'num_threads' in kwargs:
         if isinstance(kwargs['num_threads'], str):
             kwargs['num_threads'] = 'all_cpus'
 
+    if 'crs' in kwargs:
```

Comment:
This is the main change. I.e., translating input CRS codes to `pyproj.CRS` classes that can support any EPSG code.